department-of-veterans-affairs/va.gov-cms
# VACMS-4510 4557 4580 VC's

## Description
See #4510 #4557 #4580 #4624 #4625 #4626 #4627

Apologies for lumping these together, @kevwalsh - worried that if I didn't, I would potentially create a mess / not be able to get them all in before end of sprint.

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/110708935-70741f00-81c9-11eb-8976-6be0bdb28cd7.png)
![image](https://user-images.githubusercontent.com/2404547/110707595-85e84980-81c7-11eb-9947-4e0cc81346fb.png)
![image](https://user-images.githubusercontent.com/2404547/110707609-8a146700-81c7-11eb-84a9-1f67325ac48c.png)
![image](https://user-images.githubusercontent.com/2404547/110707623-8e408480-81c7-11eb-88d7-0e35ac319327.png)

## QA steps
As an administrator, go to `admin/structure/types/manage/vet_center_mobile_vet_center/form-display` & `admin/structure/types/manage/vet_center_mobile_vet_center/display` & visually verify that `Operating status - INACTIVE` & `Operating status more info - INACTIVE` are disabled
Go to `node/16478/edit` and visually verify that selecting nearby vet centers brings up a modal with vet centers that can be selected.
Go to `/admin/structure/types/manage/vet_center_locations_list/auto-label` and visually verify auto label is set to "Locations"

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
